decision_tree_ensembles/tree_ensemble_example.ipynb

```diff
@@ -268,176 +249,45 @@
-       "      <th>recall_test</th>\n",
-       "      <th>f1_test</th>\n",
-       "    </tr>\n",
-       "  </thead>\n",
-       "  <tbody>\n",
-       "    <tr>\n",
-       "      <th>DecisionTreeClassifier</th>\n",
-       "      <td>1.0</td>\n",
-       "      <td>1.0</td>\n",
-       "      <td>1.0</td>\n",
-       "      <td>0.952</td>\n",
-       "      <td>0.896</td>\n",
-       "      <td>0.923</td>\n",
-       "    </tr>\n",
-       "    <tr>\n",
-       "      <th>RandomForestClassifier</th>\n",
-       "      <td>1.0</td>\n",
-       "      <td>1.0</td>\n",
-       "      <td>1.0</td>\n",
-       "      <td>0.985</td>\n",
-       "      <td>0.955</td>\n",
-       "      <td>0.970</td>\n",
-       "    </tr>\n",
-       "    <tr>\n",
-       "      <th>AdaBoostClassifier</th>\n",
-       "      <td>1.0</td>\n",
-       "      <td>1.0</td>\n",
-       "      <td>1.0</td>\n",
-       "      <td>0.956</td>\n",
-       "      <td>0.970</td>\n",
-       "      <td>0.963</td>\n",
-       "    </tr>\n",
-       "    <tr>\n",
-       "      <th>GradientBoostingClassifier</th>\n",
-       "      <td>1.0</td>\n",
-       "      <td>1.0</td>\n",
-       "      <td>1.0</td>\n",
-       "      <td>0.970</td>\n",
-       "      <td>0.970</td>\n",
-       "      <td>0.970</td>\n",
-       "    </tr>\n",
-       "  </tbody>\n",
-       "</table>\n",
-       "</div>"
-      ],
-      "text/plain": [
-       "                            precision_train  recall_train  f1_train  \\\n",
-       "DecisionTreeClassifier                  1.0           1.0       1.0   \n",
-       "RandomForestClassifier                  1.0           1.0       1.0   \n",
-       "AdaBoostClassifier                      1.0           1.0       1.0   \n",
-       "GradientBoostingClassifier              1.0           1.0       1.0   \n",
-       "\n",
-       "                            precision_test  recall_test  f1_test  \n",
-       "DecisionTreeClassifier               0.952        0.896    0.923  \n",
-       "RandomForestClassifier               0.985        0.955    0.970  \n",
-       "AdaBoostClassifier                   0.956        0.970    0.963  \n",
-       "GradientBoostingClassifier           0.970        0.970    0.970  "
-      ]
+      "text/plain": "                            precision_train  recall_train  f1_train  \\\nDecisionTreeClassifier                  1.0           1.0       1.0   \nRandomForestClassifier                  1.0           1.0       1.0   \nAdaBoostClassifier                      1.0           1.0       1.0   \nGradientBoostingClassifier              1.0           1.0       1.0   \n\n                            precision_test  recall_test  f1_test  \nDecisionTreeClassifier               0.952        0.896    0.923  \nRandomForestClassifier               0.985        0.955    0.970  \nAdaBoostClassifier                   0.956        0.970    0.963  \nGradientBoostingClassifier           0.970        0.970    0.970  ",
```

Comment:
i'm not sure why my pycharm/jupyter reformatted this. the previous formatting was more readable. but i wanted to be sure the correctness of the output was the same, so i included it here.